start-mycroft.sh

```diff
@@ -179,81 +183,90 @@ function check-dependencies() {
 
 _opt=$1
 _force_restart=false
+
+if [ $# -eq 0 ]; then
+	help
+	return
+fi
+
 shift
-if [[ "${1}" == "restart" ]] || [[ "${_opt}" == "restart" ]] ; then
+if [ "${1}" = "restart" ] || [ "${_opt}" = "restart" ] ; then
     _force_restart=true
-    if [[ "${_opt}" == "restart" ]] ; then
+    if [ "${_opt}" = "restart" ] ; then
         # Support "start-mycroft.sh restart all" as well as "start-mycroft.sh all restart"
         _opt=$1
     fi
-    shift
+
+    if [ $# -gt 1 ]; then
```

Comment:
I think we want this as `-gt 0` as the first param has already been shifted off.